docker-compose.yml

```diff
@@ -65,30 +64,13 @@ services:
       - db
       - redis
 
-  # Worker: is a celery based worker process that runs in the background
+  # TaskIQ worker
   worker:
     container_name: ${PROJ_NAME}-worker
     build:
       context: .
       dockerfile: Dockerfile
-    command: ["watchmedo", "auto-restart", "--directory=./", "--pattern=*.py", "--recursive", "--", "celery", "--app=${PROJ_NAME}.celery.app", "worker", "--pool=gevent", "--loglevel=INFO", "--queues=celery"]
-    env_file:
-      - .env.development
-    restart: unless-stopped
-    volumes:
-      - ./src/${PROJ_NAME}:/opt/${PROJ_NAME}
-    depends_on:
-      - db
-      - redis
-
-  # Beta is Celery's cron like feature which keepts taps on
-  # repeating tasks defined by the application
-  beat:
```

Comment:
If you want to support task scheduler, you can add it using taskiq-scheduler. 

https://taskiq-python.github.io/guide/scheduling-tasks.html

Reply:
Thank you for that. I have resolved this and have a production configuration as recommended by the project.